web/src/i18n/translations/ja.yml

```diff
@@ -1 +1,83 @@
-Create a Workspace: Create a Workspace
+Personal Account: ''
+Teams: ''
+new workspace: ''
+Account Settings: ''
+Logout: ''
+Re:Earth CMS: ''
+Project name: ''
+Please input the name of project!: ''
+Project description: ''
+Accessibility: ''
+Settings: ''
+Workspace name: ''
+Please input the title of the current workspace!: ''
+Member: ''
+Account: ''
+Integration: ''
+Role: ''
+API key: ''
+Welcome to Re:Earth CMS !: ''
+No Projects Yet: ''
+Create a new project: ''
+New Project: ''
+Or read: ''
+how to use Re:Earth CMS: ''
+first: ''
+Add member: ''
+Email address or user name: ''
+Role Settings: ''
+Please input the appropriate role for this member!: ''
+select role: ''
+Owner: ''
+Writer: ''
+Reader: ''
+Add Field: ''
+Create: ''
+Setting: ''
+Display name: ''
+Please input the display name of field!: ''
+Field Key: ''
+Please input the key of field!: ''
+Description: ''
+Set Options: ''
+Please input value or delete this field.: ''
+Option value: ''
+New: ''
+Stores a list of values instead of a single value: ''
+Support multiple values: ''
+Prevents saving an entry if this field is empty: ''
+Make field required: ''
+Ensures that a multiple entries can't have the same value for this field: ''
+Set field as unique: ''
+Default value: ''
+Set default value: ''
+Asset: ''
+Update: ''
+Validation: ''
+Set maximum length: ''
+Set minimum value: ''
+Set maximum value: ''
+Model name: ''
+Please input the name of the model!: ''
+Model description: ''
+Model key: ''
+Please input the key of the model!: ''
+Key is not valid: ''
+Schema: ''
+Models: ''
+Add: ''
+input search text: ''
+Create a Workspace: ''
+Are you sure you want to delete this project?: ''
+Name: ''
+Write something here to describe this record.: ''
+Save changes: ''
+Danger Zone: ''
+Delete project: ''
+Are you sure to remove this member?: ''
+Remove this member from workspace means this member will not view any content of this workspace.: ''
+Change Role?: ''
+Remove: ''
+Members: ''
+New Member: ''
+Loading: ''
```

Comment:
```suggestion
Set Options: オプションを設定
Please input value or delete this field.: 値を入力するかこのフィールドを削除してください
Option value: オプションの値
New: 追加
Stores a list of values instead of a single value: このフィールドに複数の値をリスト形式で設定します
Support multiple values: 複数値の設定を許可
Prevents saving an entry if this field is empty: このフィールドを必須項目とします
Make field required: このフィールドを必須項目にする
Ensures that a multiple entries can't have the same value for this field: 複数のアイテム間でこのフィールドの値がユニークであることを担保します
Set field as unique: ユニーク制約
Default value: デフォルト値
Set default value: デフォルト値を設定
Asset: アセット
Update: アップデート
Validation: バリデーション
Set maximum length: 最大長
Set minimum value: 最小値
Set maximum value: 最大値
Model name: モデル名
Please input the name of the model!: このモデル名を入力してください
Model description: モデル概要
Model key: モデルキー
Please input the key of the model!: このモデルのキーを入力してください
Key is not valid: 無効なキーです
Schema: スキーマ
Models: モデル
Add: 追加
input search text: 検索
Create a Workspace: 新規ワークスペース
Are you sure you want to delete this project?: 本当にプロジェクトを削除してもよろしいですか？
Name: 名前
Write something here to describe this record.: このレコードの説明を入力してください
Save changes: 変更を保存
Danger Zone: 重要操作項目
Delete project: プロジェクト削除
Are you sure to remove this member?: 本当にメンバーをこのワークスペースから削除してもよろしいですか？
Remove this member from workspace means this member will not view any content of this workspace.: メンバーをワークスペースから削除すると、そのメンバーはワークスペース内のいかなるコンテンツも閲覧することができなくなります。
Change Role?: ロールを変更
Remove: 削除
Members: メンバー
New Member: メンバーを追加
Loading: ローディング中
```